.travis.yml

```diff
@@ -2,5 +2,9 @@ sudo: required
 language: bash
 env:
   - DOKKU_VERSION=master
```

Comment:
OK, I'm testing 0.4 to 0.8 since the README claims those versions are supported.

Tests don't pass with 0.3 and the README doesn't claim support, but it did offer installation instructions for 0.3. I tagged the version last known to work on 0.3 and explained this in the README.